src/validate-record.js

```diff
@@ -0,0 +1,42 @@
+/**
+*
+* @licstart  The following is the entire license notice for the JavaScript code in this file.
+*
+* Shared modules for microservices of Melinda record batch import system
+*
+* Copyright (C) 2018-2019 University Of Helsinki (The National Library Of Finland)
+*
+* This file is part of melinda-record-import-transformer-helmet
+*
+* melinda-record-import-transformer-helmet program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* melinda-record-import-transformer-helmet is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Affero General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*
+* @licend  The above is the entire license notice
+* for the JavaScript code in this file.
+*
+*/
+
+export default function (callback) {
```

Comment:
Remove this file and add implementation to `validate.js`. Now that validation is done only inside the actual transformer, we can instantiate validators in the same module:
```js
export default async ({fix = true, validateFixes = true}) => {
  const validate = validateFactory([
    // Instantiate validators...
    //await FieldsPresent([/^(020|022|024)$/]),
  });

  return async records => {
    const results =  await Promise.all(records.map(r => validate(r, {fix, validateFixes})));

    return results.map(({record, valid, report}) => ({
      record,
      failed: valid === false,
      messages: report
    });
  };
}
```